spec/poros/review_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
```

Comment:
Might be easier to use VCR instead of having that entire content listed there but it ultimately still works.

Edit: discussed in pair, need to check about VCR creating cassettes for every test.